serokell-style.md

```diff
@@ -521,7 +573,12 @@ foo = do
 
 ### ApplicativeDo
 
-TODO: use everywhere if possible?
+Don't use `-XApplicativeDo` blindly everywhere. Only when necessary.
```

Comment:
Why? I love it.

Reply:
Mostly because not much people clearly understand what `-XApplicativeDo` and how. Desugaring rules are less trivial. It's not clear, when monadic desugaring will be performed and when applicative. Whether there drawbacks or not? You can see strange errors with your code when everything is ok in monadic `do`-notation but doesn't compile in applicative. TL;DR this mechanism is rather new, not enough best practices, sometimes it's hard to fix errors, etc.